src/components/Guestbook/Guestbook.jsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { useState } from 'react';
+import { useEntries } from '../../context/EntryContext';
+import { useUser } from '../../context/UserContext';
+//form
+export default function Guestbook() {
+  const [name, setName] = useState('');
+  const [newEntry, setNewEntry] = useState('');
+  const { entries, setEntries } = useEntries();
+  const { user, setUser } = useUser();
+
+  function guestName() {
+    if (!newEntry) return;
+    setUser(name);
+    setEntries([...entries, { name, message: newEntry }]);
+    setNewEntry('');
+  }
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    guestName();
+  };
+
+  const guestNameInput = (
+    <div>
+      <div>
+        <label>Guest Name</label>
```

Comment:
this label can wrap the input, and the divs that start on line 26 and line 29 can be deleted for cleaner, less div soupy code